lib/components/HeroCard/HeroCard.tsx

```diff
@@ -97,23 +97,29 @@ export const HeroCard = ({
         className="ds-flex ds-rounded-lg ds-outline-none ds-transition-transform hover:ds-scale-105 focus:ds-scale-105"
       >
         <div
-          className="ds-relative ds-flex ds-flex-row sm:ds-flex-col items-center sm:ds-items-start ds-grow ds-gap-3 sm:ds-gap-6 ds-rounded-lg ds-p-5 sm:ds-p-7 sm:ds-pb-6 ds-tracking-widest ds-justify-between"
+          className="ds-relative ds-flex ds-flex-row sm:ds-flex-col ds-items-start ds-grow ds-gap-3 sm:ds-gap-6 ds-rounded-lg ds-p-5 sm:ds-p-7 sm:ds-pb-6 ds-tracking-widest ds-justify-between"
```

Comment:
Should these still be like `ds-items-center sm:ds-items-start`? I see that it previously missed the `ds-`, but in the design Layouts, the arrow is still in the center vertically.